pkg/scrape/wankz.go

```diff
@@ -74,8 +77,11 @@ func WankzVRSite(wg *sync.WaitGroup, updateSite bool, knownScenes []string, out
 			size = "1280"
 		}
 		for _, x := range []string{"1", "2", "3", "4", "5", "6"} {
-			tmpGallery := "https://cdns-i." + scraperID + ".com/" + sc.SiteID[0:1] + "/" + sc.SiteID[0:4] + "/" + sc.SiteID + "/thumbs/" + size + "_" + x + ".jpg"
-			sc.Gallery = append(sc.Gallery, tmpGallery)
+		if scraperID == "tranzvr" {
+			continue //TranzVR does no longer has preview images
+		}
```

Comment:
Instead of skipping loop iterations, wrap the entire for-loop in an if-statement.

Reply:
fixed with "break". thanks